# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1325477387 28800
#      Sun Jan 01 20:09:47 2012 -0800
# Branch stable
# Node ID 09cc39ba83120b764d6bed33df20958d70d3be1e
# Parent  c3309e1ec50d627ec3b1e237d30a8f2f1fa88bd1
test_contin.m: Rename script to test_line_continue.m for clarity

* test_contin.m: Rename script to test_line_continue.m for clarity

diff --git a/test/test_contin.m b/test/test_line_continue.m
rename from test/test_contin.m
rename to test/test_line_continue.m
